uber/templates/emails/emails_by_interest.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base-admin.html" %}
+{% block title %}Email addresses by interest{% endblock %}
+{% block content %}
+
+    <h1>Emails by Interest</h1>
+
+    Use this page to generate a CSV file of emails of people interested in hearing about a particular area of
+    {{ c.EVENT_NAME }}.  <br/><br/>
+
+    <form action="emails_by_interest_csv">
```

Comment:
Fun fact: if you wanted, you could say ``action="emails_by_interest.csv"``, because CherryPy automatically converts ``.`` to ``_`` characters in URLs.

No actual reason to change this, but I figured I'd point it out in case people didn't know it.